docs/reference/primitives/app/service.md

```diff
@@ -164,6 +167,29 @@ services:
 | --------- | ------ | ------- | ------------------------------------------------------------------------------------------ |
 | **cpu**     | number |         | The percentage of CPU utilization to target for [Processes](/reference/primitives/app/process) of this Service    |
 | **memory**  | number |         | The percentage of memory utilization to target for [Processes](/reference/primitives/app/process) of this Service |
+| **external**  | array of object |         | The array of the external metrics based on which it will scale the Service |
+
+### scale.targets.[]external
+
+| Attribute | Type   | Default | Description                                                                                |
+| --------- | ------ | ------- | ------------------------------------------------------------------------------------------ |
+| **name**     | string |         | The name of the metric |
+| **matchLabels**  | map |         | Key value lablels for the metrics |
+| **averageValue**  | number |         | The target value of the average of the metric across all relevant pods |
+| **value**  | number |         | The target value of the metric |
+
+```yaml
```

Comment:
Would be amazing to have another example of this, maybe using the AWS k8s Cloudwatch adapter? https://aws.amazon.com/blogs/compute/scaling-kubernetes-deployments-with-amazon-cloudwatch-metrics/

Reply:
Since this enables external metrics support for service scaling, so AWS k8s Cloudwatch adapter will also work.
Yeah I agree example with would be amazing.
I think it's better have a blog like example for this
cc: @heronrs

Reply:
That's a great idea nahid. I'll add a ticket